code_review_guide.md

```diff
@@ -0,0 +1,66 @@
+# Code Review Guide
+
+## Things to do before you start reviewing the PR
+
+* Make sure you are familiar with the packages the PR modifies.
+
+* Make sure you have enough continuous time to review the PR, use 300 LOC per hour to estimate.
+
+* Make sure you can follow the updates of the PR in the next few work days.
+
+* Read the description of the PR, if it's not easy to understand, ask the coder to improve it.
+
+* For a bug fix PR, if there is no test case, ask the coder to add tests.
+
+* For a performance PR, if no benchmark result is provided, ask the coder to add a benchmark result.
+
+
+## Things to check during the review process
+
+* Am I able to understand the purpose of each unit test?
+
+* Do unit tests actually test that the code is performing the intended functionality?
+
+* Do unit tests cover all the important code blocks and specially handled errors?
+
+* Could procedure tests be rewritten to table driven tests?
+
+* Is the code written following the style guide?
+
+* Is the same code duplicated more than twice?
```

Comment:
Does some code appear more than on place? Usually this can be refactor as a function.